releasenotes/notes/SNMP-Traps-Support-v1-and-v3-7ea83c24e9c37dff.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    SNMP Trap Listener now also supports v1 and v3 on top of the existing v2 support.
```

Would it make sense to expand it a little bit like this?
```suggestion
    SNMP Trap Listener now also supports protocol versions 1 and 3 on top of the existing v2 support.
```